### PR TITLE
Update importer name to include 7.x

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

**Comment by rbioteau** [comment]
Please update SWTBot dependencies

File: tests/org.bonitasoft.studio.configuration.test/src/org/bonitasoft/studio/configuration/test/swtbot/TestConfigurationDialog.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public void describeTo(final Description description) {
 
 	@Test
 	public void importDiagramWithCustomActorMappingAndOpenConfiguration() throws IOException{
-	    SWTBotTestUtil.importProcessWIthPathFromClass(bot, "TestImport-1.0.bos", "Bonita 6.x", "TestImport (1.0)", this.getClass(), false);
+	    SWTBotTestUtil.importProcessWIthPathFromClass(bot, "TestImport-1.0.bos", "Bonita 6.x and 7.x", "TestImport (1.0)", this.getClass(), false);
```
Comment (rbioteau) [inline tests/org.bonitasoft.studio.configuration.test/src/org/bonitasoft/studio/configuration/test/swtbot/TestConfigurationDialog.java]: Extract importer name in a constant ?

Comment (apupier) [inline tests/org.bonitasoft.studio.configuration.test/src/org/bonitasoft/studio/configuration/test/swtbot/TestConfigurationDialog.java]: yes, will do it when test will be ok on CP1 and maintenance

**Comment by rbioteau** [comment]
Please remake pr on CP3

**Pull request closed without merging**